knimin/lib/data_access.py

```diff
@@ -791,7 +791,11 @@ def format_survey_data(self, md, external_surveys=None, full=False):  # noqa
             # Get rid of columns not wanted for pulldown
             if not full:
                 for col in ebi_remove:
-                    del md[1][barcode][col]
+                    try:
+                        del md[1][barcode][col]
+                    except:
+                        # Column doesn't exist already, so no removal needed
```

Comment:
Probably not useful, since the way this happens is a question is retired, and therefore not in the pulldown any more, but the EBI removal list is for all iterations of the survey. All it would tell you is the person has a newer version of the survey.